src/OSPSuite.Infrastructure.Import/Core/DataImporterSettings.cs

```diff
@@ -22,6 +22,8 @@ public class DataImporterSettings : IWithIcon
 
       public bool PromptForConfirmation { get; set; } = false;
 
+      public bool IgnoreSheetNamesAtImport { get; set; }
```

Comment:
No default?

Reply:
Well, default for bool is false... But you are right, I will add it explicitly.